lib/loader.js

```diff
@@ -1,20 +1,17 @@
-'use strict';
+const fs = require('fs');
+const debug = require('debug')('express-iroute');
 
-var fs = require('fs');
-var path = require('path');
-var debug = require('debug')('express-iroute');
-
-var dotFileMatch = /\/\.[^/]*$/;
+const dotFileMatch = /\/\.[^/]*$/;
 
 function dispatch(handlers, req, res, done) {
-  var i = 0;
-  var next = function (err) {
+  let i = 0;
+  const next = function (err) {
     if (err) {
       done(err);
       return;
     }
 
-    var handler = handlers[i++];
+    const handler = handlers[(i += 1)];
```

Comment:
looks like `handlers[(i += 1)]` behaves like `handlers[++i]` rather than `handlers[i++]`, could you help confirm?

Reply:
yeah, my bad. it should be
```
const handler = handlers[i];
i += 1
```

I was doing this because `eslint` forbids us from using `++` operator
<img width="648" alt="image" src="https://user-images.githubusercontent.com/5073227/192731338-28934f7c-95ea-4451-91a4-7c9b8b5b9cdf.png">